CONTRIBUTORS.txt

```diff
@@ -134,3 +134,5 @@ Contributors
 - Antti Haapala, 2016/01/17
 - Bert JW Regeer, 2016/02/16
 - Steve Piercy, 2016/02/26
+- Josip Delic, 2016/04/01
+- Marc Rijken, 2016/04/01  
```

Comment:
This needs to be backed out and we cannot accept the work until @mrijken submits a commit adding his own name to the history. The simplest solution from here is that he may offer a pull request to your branch in your fork and it will appear here when you merge it.